pocketoptionapi_async/websocket_client.py

```
@@ -616,36 +626,40 @@ async def _handle_disconnect(self) -> None:
-                return parts.replace('api-', '').upper()
-            elif 'demo' in parts:
-                return 'DEMO'
+            parts = url.split("//")[1].split(".")[0]
+            if "api-" in parts:
+                return parts.replace("api-", "").upper()
+            elif "demo" in parts:
+                return "DEMO"
             else:
-                return 'UNKNOWN'
+                return "UNKNOWN"
         except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)